DeAceroDLabs/marketplace-frontend
# fix: [DMP-696] Fix to click on carousel images.

## What is done

[[DMP-696] click on little carousel images](https://deacero.atlassian.net/browse/DMP-696)

## How is done

- Added object with url by clicking the little carousel images.
- Added condition to not show the little images if there is only one image.


## How to test

1. Run the project via `npm start`.
2. Click on any product.
3. Click any little image (if the product have more than 1 images)
5. You should see the big/large image changes.

## Evidence

Before:
it is not visible but before the large image did not update when clicking to the smaller images.
![image](https://user-images.githubusercontent.com/81311646/200658558-7217682d-5372-4739-a214-67570589633e.png)

This is visible:
![image](https://user-images.githubusercontent.com/81311646/200658774-4ae00a26-b3c2-4375-adad-15390b8b97d0.png)

After:

![image](https://user-images.githubusercontent.com/81311646/200658847-49d7f1bf-99b1-401b-95f7-6a16c4e97b92.png)
